### PR TITLE
Updates for Sessions API MVP

### PR DESCRIPTION
**Pull request by michael-giorgaras-cko**
[TDS2-1876](https://checkout.atlassian.net/browse/TDS2-1876)

Updates to reflect the latest agreement for Sessions API MVP

### CODE REVIEW

File: spec/components/schemas/Sessions/CardSessionSource.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿type: object
+description: The source of the payment
+discriminator:
+  propertyName: type
+  mapping:
+    token: '#/components/schemas/01_PaymentRequestTokenSource'
```
Comment (helder-goncalves-cko) [inline spec/components/schemas/Sessions/CardSessionSource.yaml]: Do you mean `network_token`?

File: spec/components/schemas/Sessions/AuthenticationType.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿type: string
+enum:
+  - regular
```
Comment (helder-goncalves-cko) [inline spec/components/schemas/Sessions/AuthenticationType.yaml]: where `regular` coming from?

Comment (michael-giorgaras-cko) [reply spec/components/schemas/Sessions/AuthenticationType.yaml]: That's like non-recurring etc. Is it meant to be called something else?

Comment (helder-goncalves-cko) [reply spec/components/schemas/Sessions/AuthenticationType.yaml]: nah makes sense :)

File: spec/components/schemas/Sessions/CardSessionSource.yaml (hunk 1 above)
Comment (michael-giorgaras-cko) [reply spec/components/schemas/Sessions/CardSessionSource.yaml]: Good catch. Pointed to `network_token` instead

**Pull request merged**